Patches/MoreMechanoids/ThingDefs_Races/Races_Mechanoid.xml

```diff
@@ -179,6 +173,10 @@
                 <li Class="PatchOperationAdd">
                     <xpath>Defs/ThingDef[defName="Mech_Flamebot"]/statBases</xpath>
                     <value>
+                        <CarryWeight>50</CarryWeight>
```

Comment:
I don't know that it matters, but the weight/bulk capacity is pretty high for a mech that looks to be about the size of a Spelopede or small dog.